packages/components/src/components/dialog/dialog.tsx

```diff
@@ -26,7 +26,7 @@ type TDialog = {
     onConfirm: () => void;
     onEscapeButtonCancel?: () => void;
     portal_element_id?: string;
-    title?: string;
+    title?: React.ReactNode;
```

Comment:
This helps to resolve type issue when common-store error is passed to ErrorComponent and further to this Dialog. Also, this title is rendered directly in jsx, so I believe that this change will facilitate future replacement of localizes, which are passed as titles to Dialog, with Localize components.